lib/smartsheet/client.rb

```diff
@@ -33,6 +68,25 @@ class Client
                 :sheets, :sights, :templates, :token, :update_requests, :users, :webhooks,
                 :workspaces
 
+    # @param token [String] access token for the API; if nil or empty, uses environment variable
+    #   `SMARTSHEET_ACCESS_TOKEN`
+    # @param assume_user [String] the email address of the user to impersonate; only available for
+    #   admin roles
+    # @param json_output [Boolean] when true, endpoints return raw JSON strings instead of hashes
+    # @param max_retry_time [Fixnum] overrides the maximum number of seconds during which eligible
+    #   errors will be retried
+    # @param backoff_method [Proc] overrides the backoff calculation method, accepting the index of
```

Comment:
Could you mention that the backoff method can return `:stop` to stop retrying immediately?